third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -830,6 +853,12 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.PrivateClusterConfig = expandPrivateClusterConfig(v)
 	}
 
+<% unless version == 'ga' -%>
+	if v, ok := d.GetOk("database_encryption"); ok {
```

Comment:
should expand be called in update as well?

Reply:
Fields are `ForceNew`, so it's not necessary